cloudvolume/datasource/precomputed/mesh/__init__.py

```diff
@@ -16,8 +17,12 @@ def __new__(cls, meta, cache, config, readonly=False):
     mesh_meta = PrecomputedMeshMetadata(meta, cache)
 
     if mesh_meta.is_sharded():
-      print(red("This mesh volume is sharded. Mesh shard support is coming but is not available yet."))
-      # return ShardedLegacyPrecomputedMeshSource(mesh_meta, cache, config, readonly) 
+      if mesh_meta.info['sharding']['@type'] == 'neuroglancer_uint64_sharded_v1':
+        return ShardedMultiLevelPrecomputedMeshSource(mesh_meta, cache, config, readonly) 
+      else:
+        # @type = neuroglancer_legacy_mesh 
+        raise UnsupportedFormatError(red("This mesh volume is sharded. Legacy sharding is not available yet."))
```

Comment:
I'm not sure how to handle this case because raising an error means the images and skeletons are also inacessible.

Reply:
I had not thought about that.  Maybe we can create a legacy sharded mesh class that does nothing but throw exceptions if it's used?

Reply:
This is a much better idea!

Reply:
I created a placeholder UnshardedMultiLevelPrecomputedMeshSource in the now-renamed multilod.py. It does little but error when you try to do anything.